### ISSUE TITLE
Current MELPA packaged version doesn't include mus directory

### ISSUE THREAD

**Issue by lvh** [TeMPOraL/nyan-mode#20]
```
~: cd .emacs.d/elpa/nyan-mode-20150128.1218/                                                                                                                                                        15:44:45
~/.e/e/nyan-mode-20150128.1218: ls                                                                                                                                                                  15:44:50
img                    nyan-mode-autoloads.el nyan-mode-pkg.el       nyan-mode.el           nyan-mode.elc
```

**Comment by quazgar** [comment]
Seems to be fixed?

    /home/xxx/.emacs.d/elpa/nyan-mode-20150128.1218/mus:
    total used in directory 432 available 173809504
    drwxr-xr-x 2 daniel daniel   4096 Aug 15 10:27 .
    drwxr-xr-x 4 daniel daniel   4096 Aug 15 10:27 ..
    -rw-r--r-- 1 daniel daniel 433006 Aug 15 10:27 nyanlooped.mp3

**Comment by TeMPOraL** [comment]
Seems to include it now.
